README.md

```diff
@@ -206,29 +206,33 @@ Read [build instructions and common build issues](BUILD_FROM_SOURCE.md).
 
 ## Testing
 
-1. Download the test scenes from this [link](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally.
+1. Run our python data download utility to retrieve the test assets:
+   ```bash
+   python -m habitat_sim.utils.datasets_download --uids habitat_test_scenes --data-path /path/to/data/
+   ```
 
 1. **Interactive testing**: Use the interactive viewer included with Habitat-Sim
    ```bash
    # ./build/viewer if compiling locally
    habitat-viewer /path/to/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb
    ```
    You should be able to control an agent in this test scene.
-   Use W/A/S/D keys to move forward/left/backward/right and arrow keys to control gaze direction (look up/down/left/right).
+   Use W/A/S/D keys to move forward/left/backward/right and arrow keys or mouse to control gaze direction (look up/down/left/right).
    Try to find the picture of a woman surrounded by a wreath.
    Have fun!
 
-1. **Physical interactions**: If you would like to try out habitat with dynamical objects, first download our pre-processed object data-set from this [link](http://dl.fbaipublicfiles.com/habitat/objects_v0.2.zip) and extract as `habitat-sim/data/objects/`.
+1. **Physical interactions**: If you would like to try out habitat with dynamical objects using the interactive viewer:
+   First setup the test object assets by running the data download utility:
+   ```bash
+   python -m habitat_sim.utils.datasets_download --uids habitat_example_objects --data-path /path/to/data/
```

Comment:
I guess users must download to `data/objects` because that's the path you specify later for `object-dir`.

Reply:
Overall, it feels a little dissatisfying to always say `/path/to/data/` and leave it up to the user. A n00b user is usually not in a position to make a good decision on where to put data (inside the hab-sim root folder? inside the hab-lab root folder? Somewhere outside both of these folders?)

Some of your changes in this PR are changing clear recommended data locations (`path/to/habitat-sim/data/objects/`) to the more ambiguous `/path/to/data/`.

Reply:
Ideally, users with a standard install provide either no path (defaults to `github_root/data/`) or directly provide `/path/to/habitat-sim/data/`. The script takes care of placing the symlinks in `data/objects/` and in both of these cases tests and examples should proceed correctly. However, users with a more complex setup will need to install the data and then manually point the simulator to their data directory or create necessary symlinks.